tf/cloudwatch.tf

```diff
@@ -0,0 +1,3 @@
+resource "aws_cloudwatch_log_group" "hiit_tribe" {
```

Comment:
we need to update this from `hiit_tribe`, right?

Reply:
The whole terraform directory was not meant to be in this project at all, that's a mistake. It's been removed.